onelogin_aws_cli/__init__.py

```diff
@@ -6,39 +6,15 @@
 import configparser
 import getpass
 import os
-import xml.etree.ElementTree as ET
+import xml.etree.ElementTree as ElementTree
 
 import boto3
 from onelogin.api.client import OneLoginClient
 
-CONFIG_FILENAME = ".onelogin-aws.config"
-
-
-def user_choice(question, options):
-    """
-    Prompt a user with a question and a specific set of possible responses
-    :param question: Specifying context for the user to select an option
-    :param options: A list of options for the user to select from
-    :return:
-    """
+from onelogin_aws_cli.configuration import Section
 
-    print(question + "\n")
-    option_list = ""
-    for i, option in enumerate(options):
-        option_list += ("{}. {}\n".format(i + 1, option))
-    selection = None
-    while selection is None:
-        print(option_list)
-        choice = input("? ")
-        try:
-            val = int(choice) - 1
-            if val in range(0, len(options)):
-                selection = options[val]
-            else:
-                print("Invalid option")
-        except ValueError:
-            print("Invalid option")
-    return selection
+CONFIG_FILENAME = ".onelogin-aws.config"
+DEFAULT_CONFIG_PATH = os.path.expanduser("~/{}".format(CONFIG_FILENAME))
```

Comment:
os.path.join?

Reply:
Right now, we only store in the home directory, and it's about making sure this still works on windows (I hope). If I use os.path.join, I still have to go find the home dir.

https://stackoverflow.com/questions/4028904/how-to-get-the-home-directory-in-python

Reply:
Just to clarify, were you suggesting I use `os.path.join`?

Reply:
Yeah, I think it'd be safer on windows to do
os.path.join(os.path.expanduser("~"), CONFIG_FILENAME)